CmsManager/CmsManagerInterface.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\DashboardBundle\CmsManager;
+
+use Sonata\BlockBundle\Model\BlockInterface;
+use Sonata\DashboardBundle\Model\DashboardInterface;
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * The CmsManagerInterface class is in charge of retrieving the correct dashboard.
```

Comment:
BTW, can you explain me how i can use the styleCI patch to modify individually the 5 first commits ?


This one is going to be complicated…

```
git apply patch.txt
git add -p
```

`git add --p` will present you with blocks of code. Type 'y' to prepare the fix commit for commit 1, then git commit to create it. Rinse and repeat until you are in this situation : 

```
commit1
commit2
commit3
fix commit1
fix commit2
fix commit3
```

Then use `git rebase -i  1146cdb` to fixup fix commits with the commit they are meant to fix.